Микросетки на флексах для всех страниц

ArkadiyOrlov opened the pull request:
---
:mortar_board: [Микросетки на флексах для всех страниц](https://up.htmlacademy.ru/htmlcss/37/user/1353497/tasks/12)

:boom: https://htmlacademy-htmlcss.github.io/1353497-sedona-37/6/

ArkadiyOrlov committed:
Добавляет микросетки на флексах для всех страниц

fyl1 commented:
![image](https://github.com/htmlacademy-htmlcss/1353497-sedona-37/assets/38299430/bec041f1-5119-458f-8745-51772d673619)
вылазит за пределы блока

fyl1 commented:
![image](https://github.com/htmlacademy-htmlcss/1353497-sedona-37/assets/38299430/5e5978fa-718c-49e2-8acb-400bb2c20ad1)
уплывают поля

fyl1 commented:
![image](https://github.com/htmlacademy-htmlcss/1353497-sedona-37/assets/38299430/c20b0b46-4c0e-4aa1-a7ed-d243ec9c0af1)
ссылка на следующую страницу

fyl1 commented:
![image](https://github.com/htmlacademy-htmlcss/1353497-sedona-37/assets/38299430/39e063c3-43b7-44d4-994c-a921a5e80a95)
поставить мин виз

fyl1 commented:
![image](https://github.com/htmlacademy-htmlcss/1353497-sedona-37/assets/38299430/0be3710d-f89c-4789-aa70-4808b720f606)

fyl1 commented:
добавить классы, вложить все в одну форму

ArkadiyOrlov committed:
Исправляет замечания после проверки

1.Исправлены стили кнопки "Хочу сюда"
2. Исправлены поля иконок
3. Добавлены ссылки для перехода на "Главную" и "Каталог"
4. Добавлены min-width в стилях пагинации
5. Переделан блок фильтра каталога
